README.md

```diff
@@ -18,6 +18,7 @@ Launch a fully-featured virtual TDP Hadoop cluster with a single command _or_ cu
 - The Python package `jmespath` (an Ansible dependency for JSON parsing/querying)
 - Unzip (to execute the setup scripts)
 - `jq` required to execute helper script
+- `realpath` utility
```

Comment:
Is this used on the Ansible control node? If it is a requirement of nessie it should be installed in the tdp-collection-extras nessie role.

You can also remove the dependency and use a bash command like in lines 9 and 10 of [the setup.sh](https://github.com/TOSIT-IO/tdp-getting-started/blob/master/scripts/setup.sh)

Reply:
It is actually a requirement for the setup.sh script - https://github.com/TOSIT-IO/tdp-getting-started/blob/master/scripts/setup.sh#L10

On my mac it didn't work until I installed `realpath`.

Tell me if I need to exclude this line from this PR and create another one.